run.sh

```diff
@@ -191,7 +190,7 @@ generate_distribution_load() {
 }
 
 get_rps() {
-    closed_loop_result_dir=$(echo "${result_dir}" | sed "s|${distribution}|closed|g")
+    closed_loop_result_dir=$(echo "${result_dir}" | sed "s|-${distribution}|-closed|g")
```

Comment:
This was the path grep bug that was getting a weird path name

Reply:
Well this wasn't a bug previously right? More so just an inconvenience. Does this handle the following situation?

Say I ran a job mix in closed loop and submitted the order of jobs as `vision-resnet50-4 vision-vgg19-8 vision-alexnet-2`. Now, I want to run in point distribution with the order `vision-alexnet-2 vision-resnet50-4 vision-vgg19-8`. Technically, I've ran this job mix in closed loop, so I shouldn't have to run it again to get closed loop throughput values just because the ordering was different. Will this find the throughput values from the original job mix ordering?